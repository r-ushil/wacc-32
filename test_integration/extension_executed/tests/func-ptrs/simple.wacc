begin
  int foo() is
    return 5
  end

  int() f = foo;

  int x = call f();

  exit x
end
