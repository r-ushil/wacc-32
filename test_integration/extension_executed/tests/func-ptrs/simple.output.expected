--------- stdout ---------
--------------------------
Exit code: 5
